VirtualReality/MRML/vtkVirtualRealityViewInteractorStyle.cxx

```diff
@@ -175,30 +175,67 @@ vtkVirtualRealityViewInteractorStyle::vtkVirtualRealityViewInteractorStyle()
 
   this->Internal = new vtkInternal();
 
-  for (int d = 0; d < vtkEventDataNumberOfDevices; ++d)
-  {
-    this->Internal->InteractionState[d] = VTKIS_NONE;
-
-    for (int i = 0; i < vtkEventDataNumberOfInputs; i++)
-    {
-      this->InputMap[d][i] = -1;
-    }
-  }
 
   this->AccuratePicker = vtkSmartPointer<vtkCellPicker>::New();
   this->AccuratePicker->SetTolerance( .005 );
   this->QuickPicker = vtkSmartPointer<vtkWorldPointPicker>::New();
 
   // Create default inputs mapping
+
+  // The "eventId to state" mapping (see call to `MapInputToAction` below) applies to right and left
+  // controller buttons because they are bound to the same eventId:
+  // - `vtk_openvr_binding_*.json` files define the "button to action" mapping
+  // - `vtkOpenVRInteractorStyle()` contructor defines the "action to eventId" mapping
+
+  this->MapInputToAction(vtkCommand::ViewerMovement3DEvent, VTKIS_DOLLY);
+  this->MapInputToAction(vtkCommand::Select3DEvent, VTKIS_POSITION_PROP);
```

Comment:
@LucasGandel  Did I correctly reproduce the intended behavior ? 

To help with the review, I also included comments below related to _Before_ and _After_ the VTK refactoring introduced in kitware/VTK@b7f02e622.